src/site/paragraphs/lists_of_links.drupal.liquid

```diff
@@ -30,3 +42,36 @@
     </div>
   {% endfor %}
 </div>
+
+{% if formattedVaParagraphs.length > 6 %}
+  <script>
+    function onShowAllTopics() {
+      const vaParagraphElements = document.querySelectorAll('[data-hidden-lists_of_links-entity-id="{{ entity.entityId }}"]');
+
+      // Escape early if we are unable to find the hidden categories.
+      if (!vaParagraphElements) {
+        console.warn('Unable to find hidden lists of links DOM nodes.');
+        return;
+      }
+
+      // Show the remaining categories.
+      vaParagraphElements.forEach(function(element) {
+        element.classList.remove('vads-u-display--none');
+      });
+
+      // Derive the show all topics button element.
+      const showAllTopicsElement = document.querySelector('[data-show-all-topics-button-lists_of_links-entity-id="{{ entity.entityId }}"]');
+      if (!showAllTopicsElement) {
+        console.warn('Unable to find "Show all topics" button, so we could not hide it.');
+        return;
+      }
+
+      // We don't need the Show all topics button anymore, so hide it.
+      showAllTopicsElement.classList.replace('vads-u-display--flex', 'vads-u-display--none');
+    }
+  </script>
+
+  <div class="vads-u-display--flex vads-u-justify-content--center vads-u-margin-top--3" data-show-all-topics-button-lists_of_links-entity-id="{{ entity.entityId }}">
+    <button class="usa-button" onclick="onShowAllTopics()" type="button">Show all topics</button>
+  </div>
```

Comment:
I don't think we will want to have all of this in an inline script tag. In particular I worry about `classlist` in [IE](https://caniuse.com/classlist) and we would also need to toggle things like `aria-hidden` for SR support

Reply:
Where else would you want it if not an inline script tag? Just wasn't sure where to put it

Reply:
Good point about IE, darn it... What do you suggest?

Reply:
If we could make use of the "form-expanding-group" like in that other snippet, we can get away without writing JS. That's what I would try to do. There are already so many JS toggles lol.

Reply:
Oh good idea!